ansible/roles/operator-pipeline/templates/openshift/tasks/validate-release-config.yml

```diff
@@ -0,0 +1,58 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
```

Comment:
Hey @johnbe11, this was supposed to be a static check that is going to be part of a static test suite that already exists as a Tekton task in our pipelines. Please change it to the python test function similarly as it is defined here: https://github.com/redhat-openshift-ecosystem/operator-pipelines/tree/main/operator-pipeline-images/operatorcert/static_tests/common.

Reply:
Ack, thanks for the notes, updating accordingly.